code/src/components/TaskList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+import { EmptyList } from './EmptyList'
+import { TaskItems } from './TaskItems'
+
+
+import './TaskList.css'
+
+export const TaskList = () => {
+  const items = useSelector((state) => state.tasks.items)
+
+  // here I think they empty page goes
+  if (items.length === 0 ) {
+    return (
+      <EmptyList />
+    )
+  }
```

Comment:
This could also be inside of the return on line 20, using a conditional rendering like so
`{items.length === 0 ? <EmptyList /> : (<ul>
      {items.map((item) => (
        <TaskItems key={item.id} item={item} />
      ))
      }
    </ul>)}`